www/CaptureVideoOptions.js

```diff
@@ -27,8 +27,6 @@ var CaptureVideoOptions = function () {
     this.limit = 1;
     // Maximum duration of a single video clip in seconds.
     this.duration = 0;
-    // Video quality parameter, 0 means low quality, suitable for MMS messages, and value 1 means high quality.
-    this.quality = 1;
```

Comment:
Can this remain set to 1?

I suspect removing this could affect Android. But I have not dug into Android's code to confirm what happens internally.

With the current plugin, I suspect the following:
* Android default 1 (high)
* iOS hardcoded at medium.

Since I am looking to prepare a major release, we could just keep the default as `1` (high) to not potentially change Android and to make iOS match with Android.

This PR would introduce new features and a singular breaking change to only iOS (one platform).

Looking forward to getting this PR merged in for next major release.

Reply:
I reverted this as suggested.